system/defaults.go

```diff
@@ -0,0 +1,40 @@
+/*
+AUTHORS
+  David Sutton <david@ausocean.org>
+
+LICENSE
+  Copyright (C) 2025 the Australian Ocean Lab (AusOcean).
+
+  This is free software: you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This is distributed in the hope that it will be useful, but WITHOUT
+  ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+  or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+  License for more details.
+
+  You should have received a copy of the GNU General Public License in
+  gpl.txt. If not, see http://www.gnu.org/licenses/.
+*/
+
+package system
+
+// Default values used for camera variables.
+const (
+	DefaultAutoWhiteBalance = "cloudy"
+	DefaultBitrate          = 1500
+	DefaultContrast         = 1
+	DefaultFrameRate        = 30
+	DefaultHDR              = "auto"
+	DefaultHeight           = 1080
+	DefaultInput            = "rpicam"
+	DefaultOutput           = "RTMP"
+	DefaultRotation         = 180
+	DefaultSaturation       = 1
+	DefaultSharpness        = 1
+	DefaultWidth            = 1920
+	DefaultLogging          = "Debug"
+	DefaultMode             = ""
```

Comment:
For clarity, while the empty string is valid, `DefaultMode` should be **Normal** not an empty string.